tilavarauspalvelu/api/graphql/types/reservation/mutations.py

```diff
@@ -134,16 +134,18 @@ class Meta:
 
 
 class ReservationDeleteMutation(DeleteMutation):
```

Comment:
Should also rename this to `DeleteTentativeReservationMutation`.

Reply:
Ah, of course! 😅

Reply:
Wait, that would actually not work since it renames the mutation type, which would break frontend